src/styles/main.scss

```diff
@@ -1,3 +1,24 @@
+@import "./utilit/variables.scss";
+@import "./calendar.scss";
+
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
+  height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+html {
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+}
+
+.main {
+  display: flex;
+  justify-content: center;
 }
```

Comment:
u should create normalize.scss for * and html. also I think you need to create separate block for the body and make main its child.  main.scss only have to contain the paths to other scss files

Reply:
done